packages/lib-utils/src/components/table/Table.tsx

```diff
@@ -0,0 +1,95 @@
+// @ts-nocheck
+import * as React from 'react';
+import {
+  ICell,
+  OnSelect,
+  SortByDirection,
+  TableComposable,
+  TableGridBreakpoint,
+  Tbody,
+  // Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
```

Comment:
Did we try starting with https://www.patternfly.org/v4/components/table/ first and tweak it to what we need? I'm worried we're rebuilding another table implementation using the patternfly building blocks. That might be necessary in the end, but we should start simple.

 I think the idea is we want a `Table` component as a middle man between SDK consumers and the Patternfly table that can sprinkle some K8s handling magic inbetween.

Reply:
Not sure I'm getting your point. Do you mean using `<Table>` PF component instead of `<TableComposable>` with the build blocks? I wanted to avoid using `<Table>` since it's marked as legacy - see https://www.patternfly.org/v4/components/table/react-legacy

Reply:
Ah, I completely missed that! Good point. I didn't see the `TableComposable` import